### PR TITLE
🔸 Replaces us-election-midriff with donald-trump-midriff

### PR DESCRIPTION
**Pull request by aendrew**
Basically just renames a bunch of stuff. Also removes a lot of the election midriff graphics stuff.

screen shot 2016-11-10 at 15 59 28

### CODE REVIEW

File: server/sections/regional-news/presenter.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = function regionalNewsPresenter ({flags, region, edition}){
 	const otherItems = concat([
 		topStoriesData.topStories,
 		usElectionData.usElectionHeadpiece,
-		usElectionData.usElectionMidriff,
+		// usElectionData.trumpMidriff,
```
Comment (kavanagh) [inline server/sections/regional-news/presenter.js]: Why has this been removed?

Comment (aendrew) [inline server/sections/regional-news/presenter.js]: @kavanagh I figured I'd just replace any instance of usElectionMidriff, that was one of the ones I wasn't sure about. I'm not sure `usElectionData.usElectionMidriff` is there anymore given that I've replaced it elsewhere, though I can revert that pretty easily.

Comment (kavanagh) [inline server/sections/regional-news/presenter.js]: I don't think I should review this. I'm not sure what that does. 

How does this pertain to the Regional News section? Perhaps this is something to do with deduping.

**Comment by kavanagh** [comment]
Not sure we needed to rename everything. But I suppose it's done now.

File: server/sections/regional-news/presenter.js (hunk 1 above)
Comment (aendrew) [inline server/sections/regional-news/presenter.js]: @kavanagh That'd be my guess. Fair point though, will get somebody on the next-front-page team to take a look.

File: server/data/graphql-queries/parts/trump-midriff.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// This is still called "usElectionMidriff" because an update is
+// needed at:
+// https://github.com/Financial-Times/next-api/blob/master/server/graphql/apis/us-election-midriff.js
+
+
```
Comment (andygout) [inline server/data/graphql-queries/parts/trump-midriff.js]: Change has been made in `next-api` so can remove these comments + blank lines.

**Comment by andygout** [comment]
Cannot be merged until `next-api` has correct list value got [`trumpMidriff`](https://github.com/Financial-Times/next-api/blob/master/server/config/ids.js#L38) so making as 'changes requested' until then.

File: server/sections/regional-news/presenter.js (hunk 1 above)
Comment (GlynnPhillips) [inline server/sections/regional-news/presenter.js]: CORRECT 🎉 This seems right to me

**Pull request merged**